### PR TITLE
Docs page navigation adjustments

### PR DESCRIPTION
**Pull request by insectengine**
The docs page had problems with the on page subnav rendering with the added styles. I removed the stickiness to make it present cleaner and created a subnav pulldown for mobile sizes.

### CODE REVIEW

File: docs/_sass/position.scss [hunk 1]
```diff
@@ -179,7 +179,7 @@ $position-large-margin-l:                               50px !default;
 
 .uk-position-absolute { position: absolute !important; }
 
-.uk-position-fixed { position: fixed !important; }
+.uk-position-fixed { /* position: fixed !important; */ }
```
Comment (aureamunoz) [inline docs/_sass/position.scss:182]: Shouldn't we just remove the commented part? 
```
.uk-position-fixed { /* position: fixed !important; */ }
```

File: docs/_sass/width.scss [hunk 2]
```diff
@@ -192,7 +192,8 @@ $width-2xlarge-width:                            750px !default;
     .uk-width-2-3\@s { width: unquote('calc(100% * 2 / 3.001)'); }
 
     /* Quarters */
-    .uk-width-1-4\@s { width: 25%; }
+/*  .uk-width-1-4\@s { width: 25%; } */
+/*  .uk-width-1-4\@s { width: 25%; } */
     .uk-width-3-4\@s { width: 75%; }
```
Comment (aureamunoz) [inline docs/_sass/width.scss:197]: Same remark, can we remove the commented part or it is used for something?

**Review by aureamunoz — COMMENTED** [review]
Great Job, Thank you!!!! I don't have to much to day, just a question about the commented lines :-)

**Comment by insectengine** [comment]
Hi Aurea,

Yes you can remove. I commented it out during testing and should have removed it myself.

Regards
James

**Review by aureamunoz — APPROVED** [review]

**Pull request merged**